app-modules/authorization/tests/Feature/Filament/Resources/User/ListUsersTest.php

```diff
@@ -396,3 +396,35 @@
         ->assertCanSeeTableRecords($usersWithoutLicense)
         ->assertCanNotSeeTableRecords($usersWithRetentionCrmLicense->merge($usersWithRecruitmentCrmLicense)->merge($usersWithConversationalAiLicense));
 });
+
+it('Filter users based on Created After', function () {
+    asSuperAdmin();
+
+    $createdAfterUsers = User::factory()
+        ->count(5)
+        ->sequence(
+            ['created_at' => '2024-11-20 23:00:00'],
+            ['created_at' => '2024-11-22 03:00:00'],
+            ['created_at' => '2024-11-23 04:00:00'],
+            ['created_at' => '2024-11-24 05:30:00'],
+            ['created_at' => '2024-11-25 06:00:00'],
+        )
+        ->create();
+
+    $createdBeforeUsers = User::factory()
+        ->count(5)
+        ->sequence(
+            ['created_at' => '2024-11-20 22:00:00'],
+            ['created_at' => '2024-11-18 03:00:00'],
+            ['created_at' => '2024-11-17 04:00:00'],
+            ['created_at' => '2024-11-16 05:30:00'],
+            ['created_at' => '2024-11-15 06:00:00'],
+        )
+        ->create();
+
+    livewire(ListUsers::class)
+        ->set('tableRecordsPerPage', 6)
```

Comment:
```suggestion
        ->set('tableRecordsPerPage', 10)
```

Let's change this to 10 so we can be sure this is not pagination hiding the Users.

It can also be good to assert we see all Users before the filter is applied.

Reply:
Done